styles.css

```diff
@@ -8,7 +8,7 @@ body {
 #main {
   width: 100%;
   background-color: rgb(232, 232, 232);
-  border: 0px solid lightgray;
+  border: 1px solid lightgray;
   border-radius: 5px;
   padding-top: 20px;
   padding-bottom: 20px;
```

Comment:
I think it would be better to use Flex box (flexible box layout) that provides a flexible and efficient way to organize and align content within a container., Please try to use it in this container, because it's  better and responsive.